### DISCUSSION TITLE
Data Quality should come under analysis

### DISCUSSION THREAD

**Discussion by nairabhishek73** [DevDataPlatform/webapp#1276] [General]
@ashwinit4d @fatchat @siddhant3030 @Ishankoradia @himanshudube97 @pratzrao @nishika26 
Thoughts or comments? 
I don't think Data Quality should be a separate line item on the menu bar. Currently since its a dahsboard it should come under analysis, later when/if it becomes a part of configurable pipeline setup then it should come under pipleine.

**Comment by Ishankoradia** [comment]
I dont mind putting it under the `analysis` tab.
